Dockerfile

```diff
@@ -196,13 +213,23 @@ RUN yum -y install \
         -skip qtwebglplugin \
         -skip qtwebsockets \
         -skip qtwebview \
-        -skip qtxmlpatterns && \
-    make -s && \
+        -skip qtxmlpatterns \
+        -icu -I /opt/icu/include -L /opt/icu/lib && \
+    LD_LIBRARY_PATH=/opt/icu/lib make -s && \
     make install && \
     popd && \
     rm -rf /tmp/qt_download && \
     yum clean all
 
+# Move the ICU binaries over to the Qt libs and make sure libQt5Core finds them.
+RUN cp /opt/icu/lib/libicu* /root/qt_5_15_0/lib/ && \
+    yum -y install epel-release && \
+    yum -y install patchelf && \
+    patchelf --set-rpath '$ORIGIN' /root/qt_5_15_0/lib/libQt5Core.so.5.15.0 && \
```

Comment:
The official binaries also have this rpath setting and also have the icu libs in the `qt/lib` directory.